layouts/shortcodes/button/about-atsign.html

```diff
@@ -0,0 +1,6 @@
+<a href="#" class="btn btn-primary btn-lg mr-3 mb-4" role="button" aria-pressed="true">
+    <div style="display: flex; align-items: center;">
+        <p style="margin: 0px;font-size: medium;">What is Atsign?</p>
+        <i class="fas fa-info-circle"></i>
+    </div>
+</a>
```

Comment:
Please add a new line to this file, when you modify it again.

Reply:
Done